src/norwegianblue/cli.py

```diff
@@ -136,13 +136,17 @@ def main() -> None:
                 show_title=multiple_products,
             )
         except ValueError as e:
-            prompt = f"{e} [Y/n] "
+            suggestion = norwegianblue.suggest_product(product)
+
+            prompt = f"{e}{' [Y/n] ' if suggestion else ''}"
             if args.color != "no":
                 prompt = colored(prompt, "yellow")
+            if not suggestion:
+                print(prompt)
+                sys.exit()
             answer = input(prompt)
             if answer not in ("", "y", "Y"):
                 sys.exit()
```

Comment:
```suggestion
                print()
                continue
            answer = input(prompt)
            if answer not in ("", "y", "Y"):
                print()
                continue
```

Reply:
How about this? When the user answers "n" or receives no suggestions, the remaining list of search terms will still be handled.

Reply:
Yes, that sounds good 👍